.github/workflows/tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Tests
```

Comment:
```suggestion
name: Test batch_processing.sh
```

Reply:
Right now it's showing as this:

![Screenshot from 2021-02-01 17-22-29](https://user-images.githubusercontent.com/16181459/106525531-1d17fe00-64b2-11eb-9177-a2f75faf5cad.png)

Reply:
Oh, right. I see. But I don't really think they should sync up: https://github.com/neuropoly/spinalcordtoolbox/pull/3198/files#r568224050

:shrug: hbu?